src/lib/services/email-service.test.ts

```diff
@@ -29,17 +30,17 @@ test('Can send reset email', async () => {
 });
 
 test('Can send welcome mail', async () => {
-    const emailService = new EmailService(
-        {
+    const emailService = new EmailService({
+        email: {
             host: 'test',
-            port: 9999,
+            port: 587,
             secure: false,
-            sender: 'noreply@getunleash.ai',
             smtpuser: '',
             smtppass: '',
+            sender: 'noreply@getunleash.ai',
```

Comment:
Why did you change the port and add the sender? That seems unrelated?

Reply:
Port is completely unrelated - some tests where using one or the other, so when I changed the constructor and copied the config from one test to the other